### PR TITLE
#3669 [Manager] Move all constant and symbol to global folder

### PR DESCRIPTION
**Pull request by saivirtue**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3669)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/api/streamApi.js [hunk 1]
```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+import { KIND } from 'const';
```
Comment (konekoya) [inline ohara-manager/client/src/api/streamApi.js]: @saivirtue could you move this out from third-party deps import section?

File: ohara-manager/client/src/const.js [hunk 2]
```diff
@@ -14,7 +14,27 @@
  * limitations under the License.
  */
 
+// default group of services
 export const WORKSPACE = 'workspace';
 export const ZOOKEEPER = 'zookeeper';
 export const BROKER = 'broker';
 export const WORKER = 'worker';
```
Comment (konekoya) [inline ohara-manager/client/src/const.js]: Should we consider putting this in an object?

```js
group: {
  // group names here
}
```

Comment (konekoya) [reply ohara-manager/client/src/const.js]: Maybe we can discuss this a bit here? @eechih @wu87988622

Comment (eechih) [reply ohara-manager/client/src/const.js]: > Should we consider putting this in an object?

這樣比較清楚～ 不過，以後可能會讓 AppConext 統一管理 group，因此這些 group constant 或許用不到了 :)

**Comment by saivirtue** [comment]
> Sorry about the Git conflict..., Please fix them and merge the PR. Thanks :)

Do mention it, it's a small change :)

**Pull request merged**